### PR TITLE
Adding HTU31D sensor definition

### PR DESCRIPTION
**Pull request by borotaman**
Wippersnapper component definition for HTU31D sensor

### CODE REVIEW

File: components/i2c/htu31d/definition.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "displayName": "HTU31D",
+    "i2cAddresses": [ "0x40", "0x41" ],
+    "subcomponents": [
+    {
+      "displayName": "Temperature/Humidity Sensor",
```
Comment (brentru) [inline components/i2c/htu31d/definition.json:6]: Please make the `displayName` less than 24 characters

Our CI reported this error here: https://github.com/adafruit/Wippersnapper_Components/runs/8004594245?check_suite_focus=true#step:4:27

File: components/i2c/htu31d/definition.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      "sensorType": ["ambient-temp", "humidity"],
+      "defaultPeriod": 900
```
Comment (brentru) [inline components/i2c/htu31d/definition.json:8]: Remove this `defaultPeriod`

**Review by brentru — CHANGES_REQUESTED** [review]
Hi @borotaman - thanks for your first contribution to Adafruit WipperSnapper. I've left two things to fix before we can merge this work in.

**Comment by tyeth** [comment]
Hey @borotaman 
I'm looking at picking up adding this component, unless you wish to carry on with this?

**Comment by tyeth** [comment]
Closed by this commit https://github.com/adafruit/Wippersnapper_Components/commit/6ab32cedb78f39a6089b41475c366f9e58c9a57c

Component driver code is finished, expect it to be released next wednesday as part of Wippersnapper v1.0.0-beta73

**Pull request closed without merging**